templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -0,0 +1,247 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/crio/crio.conf"
+contents:
+  inline: |
+    # The "crio" table contains all of the server options.
```

Comment:
It looks like this is overwriting the version from the RPM?  Is it different?

My 2¢ is that it's best to have defaults live in the code, documentation for options in a man page or so.

Not objecting to this, we don't need to remove the `lgtm` but if this isn't different from the RPM version I'm not sure what we're gaining by writing it here again.

Reply:
I made sure to match this to the one coming in from the RPM. But yeah this overrides the RPM one.